ibis/expr/types/pretty.py

```diff
@@ -13,43 +14,72 @@
 
 import ibis
 import ibis.expr.datatypes as dt
+from ibis.common.exceptions import TranslationError
 
-# A console with all color/markup disabled, used for `__repr__`
-simple_console = Console(force_terminal=False)
+if TYPE_CHECKING:
+    from rich.table import Table as RichTable
 
+    from ibis.expr.types import Column, Expr, Scalar, Table
 
-def _format_nested(values):
-    interactive = ibis.options.repr.interactive
+
+def pretty_repr(expr: ibis.Expr) -> str:
```

Comment:
This function feels similar to `__interactive_rich_console__`, in a followup they might want to get unified.